### PR TITLE
Install imgaug also on the robot and not only the pc, so the donkey c…

### PR DESCRIPTION
**Pull request by DocGarbanzo**
Install `imgaug` also on the robot and not only the pc, so the donkey car can use CROP without any additional manual installation. Also added a None type check in the augmentation part. I checked on RPi that this is working.

### CODE REVIEW

File: donkeycar/pipeline/augmentations.py [hunk 1]
```diff
@@ -117,5 +117,7 @@ def create(cls, aug_type: str, config: Config) -> iaa.meta.Augmenter:
 
     # Parts interface
     def run(self, img_arr):
+        if img_arr is None:
```
Comment (Ezward) [inline donkeycar/pipeline/augmentations.py:120]: This is a good change. This is orthogonal to the changes I made to the cameras

File: setup.py [hunk 2]
```diff
@@ -50,6 +50,7 @@ def package_files(directory, strip_leading):
           'typing_extensions',
           'pyfiglet',
           'psutil',
+          'imgaug'
```
Comment (Ezward) [inline setup.py:53]: the imgaug library break the CSIC and the opencv camera on the Jetson nano.  So we should probably add this to the pi for now, but leave it off the nano.

**Review by Ezward — CHANGES_REQUESTED** [review]
The change to augmentations is good and is orthogonal to the changes I made to the cameras.  However installing imgaug on nano breaks the CSI camera, so we can add it to the pi, but not the nano.

File: donkeycar/pipeline/augmentations.py (hunk 1 above)
Comment (DocGarbanzo) [reply donkeycar/pipeline/augmentations.py:120]: I think we don't need this any longer now, b/c you fixed the camera returning `None`.

File: setup.py (hunk 2 above)
Comment (DocGarbanzo) [reply setup.py:53]: I'll change this.

**Comment by Ezward** [comment]
@DocGarbanzo This looks good.  If you can fix the conflicts then we can merge it.

File: donkeycar/pipeline/augmentations.py (hunk 1 above)
Comment (DocGarbanzo) [reply donkeycar/pipeline/augmentations.py]: Removed through rebasing

**Comment by DocGarbanzo** [comment]
Rebased on dev and bumped version

**Review by Ezward — APPROVED** [review]
Excellent.  Thanks.
